### DISCUSSION TITLE
Next Prev button for collection item

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#12639] [Feature Requests]
Hi

We have been working with big datasets recently and one thing what would make our work faster would be easy way to move to the next collection item. Also maybe to add similar thing under save (Save and open next item).

Thanks
hembo

**Comment by rijkvanzanten** [comment]
What would control what the "next" item is? Would you have to configure your preferred sorting order on collection settings? Would it mirror what you had configured on the layout for this collection? 🤔

**Reply by HemboPagi** [reply]
I guess following the default sorting or user preferences.

**Reply by benhaynes** [reply]
Layout / User Preferences won't always work... for example, a Map layout doesn't have an _order_.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
